internal/workload/podspec_updates_test.go

```diff
@@ -622,6 +623,34 @@ func TestProxyCLIArgs(t *testing.T) {
 				fmt.Sprintf("hello:world:one?port=%d&private-ip=true", workload.DefaultFirstPort),
 				fmt.Sprintf("hello:world:two?port=%d&private-ip=false", workload.DefaultFirstPort+1)},
 		},
+		{
+			desc: "global flags",
+			proxySpec: v1alpha1.AuthProxyWorkloadSpec{
+				AuthProxyContainer: &v1alpha1.AuthProxyContainerSpec{
+					SQLAdminAPIEndpoint: "https://example.com",
+					Telemetry: &v1alpha1.TelemetrySpec{
+						HTTPPort:  ptr(int32(9092)),
+						AdminPort: ptr(int32(9091)),
+					},
+					MaxConnections:  ptr(int64(10)),
+					MaxSigtermDelay: ptr(int64(20)),
+				},
+				Instances: []v1alpha1.InstanceSpec{{
+					ConnectionString: "hello:world:one",
+				}},
+			},
+			wantProxyArgContains: []string{
+				fmt.Sprintf("hello:world:one?port=%d", workload.DefaultFirstPort),
+			},
+			wantWorkloadEnv: map[string]string{
+				"CSQL_PROXY_SQLADMIN_API_ENDPOINT": "https://example.com",
+				"CSQL_PROXY_HTTP_PORT":             "9092",
+				"CSQL_PROXY_ADMIN_PORT":            "9091",
```

Comment:
Setting the admin port alone won't enable the admin server. 

See https://github.com/GoogleCloudPlatform/cloud-sql-proxy#localhost-admin-server.

Should we make admin-related changes a separate PR?

Reply:
Yes. I'll remove the ADMIN_PORT envvar from this PR and move it into another one.